### PR TITLE
refactor: helm chart

### PR DESCRIPTION
**Pull request by CL-SHLOMIKONCHA**

### CODE REVIEW

**Review by asafMasa — CHANGES_REQUESTED** [review]
Great Work!

File: helm/raster/templates/_resources.tpl [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+{{- define "route.fullname" -}}
+{{- printf "%s-%s-%s" .Release.Name .Chart.Name "route" | indent 1 }}
+{{- end }}
+
+{{/*
+Create ingress name as used by the service name label.
+*/}}
+{{- define "ingress.fullname" -}}
+{{- printf "%s-%s-%s" .Release.Name .Chart.Name "ingress" | indent 1 }}
+{{- end }}
```
[the remark this answers is not in the document]
Comment (CL-SHLOMIKONCHA) [reply helm/raster/templates/_resources.tpl:41]: fixed

File: helm/raster/templates/_resources.tpl [hunk 2]
```diff
@@ -0,0 +1,41 @@
+{{/*
+Create service name as used by the service name label.
+*/}}
+{{- define "service.fullname" -}}
+{{- printf "%s-%s" .Release.Name "service" }}
```
[the remark this answers is not in the document]
Comment (CL-SHLOMIKONCHA) [reply helm/raster/templates/_resources.tpl:5]: fixed

File: helm/raster/templates/_tplValues.tpl [hunk 3]
```diff
@@ -0,0 +1,46 @@
+{{/*
+Copyright VMware, Inc.
+SPDX-License-Identifier: APACHE-2.0
+*/}}
+
+{{/* vim: set filetype=mustache: */}}
```
[the remark this answers is not in the document]
Comment (CL-SHLOMIKONCHA) [reply helm/raster/templates/_tplValues.tpl:6]: this template has been taken from bitnami chart 
found some explanation on: https://stackoverflow.com/questions/77458970/mustache-filetype-in-helm-template-partials

File: helm/raster/templates/_tplValues.tpl [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+{{ include "common.tplvalues.merge" ( dict "values" (list .Values.path.to.the.Value1 .Values.path.to.the.Value2) "context" $ ) }}
+*/}}
+{{- define "common.tplvalues.merge" -}}
+{{- $dst := dict -}}
+{{- range .values -}}
+{{- $dst = include "common.tplvalues.render" (dict "value" . "context" $.context "scope" $.scope) | fromYaml | merge $dst -}}
+{{- end -}}
+{{ $dst | toYaml }}
+{{- end -}}
+
```
[the remark this answers is not in the document]
Comment (CL-SHLOMIKONCHA) [reply helm/raster/templates/_tplValues.tpl:39]: fixed

File: helm/raster/templates/_resources.tpl [hunk 5]
```diff
@@ -0,0 +1,41 @@
...
+Create deployment name as used by the service name label.
+*/}}
+{{- define "deployment.fullname" -}}
+{{- printf "%s-%s-%s" .Release.Name .Chart.Name "deployment" | indent 1 }}
+{{- end }}
+
+{{/*
+Create route name as used by the service name label.
+*/}}
+{{- define "route.fullname" -}}
```
[the remark this answers is not in the document]
Comment (CL-SHLOMIKONCHA) [reply helm/raster/templates/_resources.tpl]: unused, removed

File: helm/raster/templates/_resources.tpl [hunk 6]
```diff
@@ -0,0 +1,41 @@
...
+{{- printf "%s-%s-%s" .Release.Name .Chart.Name "route" | indent 1 }}
+{{- end }}
+
+{{/*
+Create ingress name as used by the service name label.
+*/}}
+{{- define "ingress.fullname" -}}
```
[the remark this answers is not in the document]
Comment (CL-SHLOMIKONCHA) [reply helm/raster/templates/_resources.tpl]: unused, removed

**Review by asafMasa — APPROVED** [review]

**Pull request merged**